.tx/config

```diff
@@ -60,6 +60,15 @@ resource_name          = IPD App
 replace_edited_strings = false
 keep_translations      = false
 
+[o:openmrs:p:bahmni:r:micro-frontend-app]
+file_filter            = micro-frontends/public/i18n/locale_<lang>.json
+source_file            = micro-frontends/public/i18n/locale_en.json
```

Comment:
If the source language change or if there could be multiple source files,should we consider making the source file dynamic or configurable as in above line(64)
@rahu1ramesh

Reply:
@deepthi-mantena To ensure better support from Transifex, it is recommended to keep the source language as English (though non-English source languages are supported). Throughout Bahmni, we have used English as the source language to keep the strings easily maintainable for developers. Transifex expects a single source file, so there is no need to make the source file definition dynamic.